### PR TITLE
Tests for the app

### PR DESCRIPTION
**Pull request by damdafayton**
- First test file includes add and remove function tests
- Second test file includes edit and update functions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,12 +14,15 @@
...
+[Live Demo Link](https://damdafayton.github.io/to-do-list/dist)
 
 
 ## Getting Started
 
-Download `\dist` folder and click `index.html`.
+- To run: download `\dist` folder and click `index.html`.
+- To test :
+    - `npm install jest --global`
+    - `$ jest`
```
Comment (TedLivist) [inline README.md:25]: - [ ] **[OPTIONAL]** I would recommend you add the correct command for actually running the tests ie `npm test`

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: INVALID CODE REVIEW REQUEST 🛑 🛑

Hi @damdafayton,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### You have submitted a project that is collaborative whereas you are the only one with commits. As required there should be commits from the other student. If you worked solo on the project then kindly mention it in the PR so the next reviewer can know.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md (hunk 1 above)
Comment (damdafayton) [reply README.md:25]: command is correct

Comment (TedLivist) [reply README.md:25]: Oh yeah. You are right. Forgot to remove that

**Comment by damdafayton** [comment]
Yes i did solo because i did progress fast on the curriculum. I will still contribute for my partner during his activity.

**Review by Meltrust — APPROVED** [review]
Hi @damdafayton,

## Wow, you did it 🎉 
Brilliant



### To highlight:

- Nice mocking localStorage✔️ 
- Test are passing✔️ 
- Sufficient tests✔️


- [ ] OPTIONAL: Some mocks are plain massive.  I recommend  just placing them on another file, and importing them 💪 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by dasileker — APPROVED** [review]
## **Hi @damdafayton,** 

## **Congratulations! 🎉 Project Approved ✅**
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 
 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**